semantic_release/hvcs/remote_hvcs_base.py

```diff
@@ -34,7 +34,7 @@ class RemoteHvcsBase(HvcsBase, metaclass=ABCMeta):
 
     DEFAULT_ENV_TOKEN_NAME = "HVCS_TOKEN"  # noqa: S105
 
-    def __init__(self, remote_url: str, *args: Any, **kwargs: Any) -> None:
+    def __init__(self, remote_url: str, *args: Any, **kwargs: Any) -> None:  # noqa: ARG002
```

Comment:
If you think we can remove `**kwargs` here, I think we could also use it on the various specific vcs systems that inherit?

I don't think just renaming to `**_kwargs` is the right fix, but that would also shut up ruff, I think?

Reply:
This is a dumb error from ruff. yes it is needed as stated above, I'm fine with using the underscore rather than a bunch of ignores.